callosum/lower/redis.py

```diff
@@ -84,6 +85,7 @@ def __init__(self, transport: RedisStreamTransport,
         _s = asyncio.shield
         await _s(self.transport._redis.xadd(
             stream_key, {b'hdr': raw_msg[0], b'msg': raw_msg[1]}))
+        print(f"Message has been sent successfully")
```

Comment:
Maybe this print statement is just for debugging.
Remove it, or use `log = logging.getLogger(__name__)` at the top of this module and `log.debug(...)`.